README.md

```diff
@@ -16,26 +16,26 @@
 </p>
 
 <p align="center">
-  <img src="./assets/previews/preview.webp" alt="Preview" />
+  <img src="https://raw.githubusercontent.com/catppuccin/unocss/refs/heads/main/assets/previews/preview.webp" alt="Preview" />
 </p>
 
 ## Previews
 
 <details>
   <summary>🌻 Latte</summary>
-  <img src="./assets/previews/latte.webp" alt="Latte Preview" />
+  <img src="https://raw.githubusercontent.com/catppuccin/unocss/refs/heads/main/assets/previews/latte.webp" alt="Latte Preview" />
 </details>
 <details>
   <summary>🪴 Frappé</summary>
-  <img src="./assets/previews/frappe.webp" alt="Frappé Preview" />
+  <img src="https://raw.githubusercontent.com/catppuccin/unocss/refs/heads/main/assets/previews/frappe.webp" alt="Frappé Preview" />
 </details>
 <details>
   <summary>🌺 Macchiato</summary>
-  <img src="./assets/previews/macchiato.webp" alt="Macchiato Preview" />
+  <img src="https://raw.githubusercontent.com/catppuccin/unocss/refs/heads/main/assets/previews/macchiato.webp" alt="Macchiato Preview" />
 </details>
 <details>
   <summary>🌿 Mocha</summary>
-  <img src="./assets/previews/mocha.webp" alt="Mocha Preview" />
+  <img src="https://raw.githubusercontent.com/catppuccin/unocss/refs/heads/main/assets/previews/mocha.webp" alt="Mocha Preview" />
```

Comment:
Any reason for changing these away from relative links? We prefer relative URLs across the organisation.

Reply:
FYI, starlight uses non-relative URLs. I changed to that way since linking the README to different directory levels makes images not resolve to correct paths. I'll try to find a way to solve this though.

Reply:
I am going to manually create an alpha release on npm to see if npmjs resolves them correctly with the current changes.

Reply:
Done.